scripts/crawler.py

```diff
@@ -24,7 +24,7 @@
         ingo = int("ingo" in e.podcast_person)
         l33tname = int("l33tname" in e.podcast_person)
         madmas = int("madmas" in e.podcast_person)
-        makefu = int("makefu" in e.podcast_person)
+        makefu = int("felix" in e.podcast_person)
```

Comment:
is this consequently done correctly for 2024?

Reply:
yes, quick validation: only those were felix was not present

this might become difficult for https://github.com/Binaergewitter/serious-bg/issues/453 as he has 2 nick names then....

Reply:
yes, quick validation: only those were felix was not present

this might become difficult for https://github.com/Binaergewitter/serious-bg/issues/453 as he has 2 nick names then....